samples/bluetooth/mesh/ble_peripheral_lbs_coex/README.rst

```diff
@@ -0,0 +1,203 @@
+
+LEDs (Common):
+   Show the OOB authentication value during provisioning if the "Push button" OOB method is used.
+
+LED 1:
+   Controlled by the Generic OnOff Server.
+
+LED 2:
+   Controlled remotely by the LED Button Service from the connected device.
+
+Configuration
+*************
+
+|config|
+
+Source file setup
+=================
+
+This sample is split into the following source files:
+
+* :file:`main.c` used to handle initialization.
+* :file:`model_handler.c` used to handle mesh models.
+* :file:`lb_service_handler.c` used to handle the LBS interaction.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/ble_peripheral_lbs_coex`
+
+.. include:: /includes/build_and_run.txt
+
+Testing Generic OnOff Server (Bluetooth mesh)
+=============================================
+
+After programming the sample to your development kit, you can test it by using a smartphone with `nRF Mesh mobile app`_ installed.
+Testing consists of provisioning the device and configuring it for communication with the Bluetooth mesh models.
+
+Provisioning the device
+-----------------------
+
+.. |device name| replace:: :guilabel:`Mesh Light`
+
+.. include:: /includes/mesh_device_provisioning.txt
+
+Configuring models
+------------------
+
+See :ref:`ug_bt_mesh_model_config_app` for details on how to configure the mesh models with the nRF Mesh mobile app.
+
+Configure the Generic OnOff Server model on the root element of the :guilabel:`Mesh and Peripheral Coex` node:
+
+* Bind the model to :guilabel:`Application Key 1`.
+
+  Once the model is bound to the application key, you can control **LED 1** on the device.
+* In the model view, tap :guilabel:`ON`/:guilabel:`OFF` (one of the Generic On Off Controls).
+  This switches the **LED 1** on the development kit on and off respectively.
+
+Testing LED Button Service (peripheral)
+=======================================
+
+After programming the sample to your development kit, test it by performing the following steps:
+
+1. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+#. Power on the development kit.
+#. Connect to the device from the nRF Connect application.
+   The device is advertising as ``Mesh and Peripheral Coex``.
+   The services of the connected device are shown.
+#. In :guilabel:`Nordic LED Button Service`, enable notifications for the :guilabel:`Button` characteristic.
+#. Press **Button 1** on the device.
+#. Observe that notifications with the following values are displayed:
+
+   * ``Button released`` when **Button 1** is released.
+   * ``Button pressed`` when **Button 1** is pressed.
+
+#. Write the following values to the LED characteristic in the :guilabel:`Nordic LED Button Service`:
+
+   * Value ``OFF`` to switch the **LED 2** on the development kit off
+   * Value ``ON`` to switch the **LED 2** on the development kit on
```

Comment:
And since we're anyways fixing a few things, add periods to these two lines as well :) They managed to slip.